package.json

```diff
@@ -23,8 +23,8 @@
     "webpack": "^1.12.9"
   },
   "scripts": {
-    "build": "./node_modules/webpack/bin/webpack.js",
-    "watch": "./node_modules/webpack/bin/webpack.js --watch",
-    "test": "./node_modules/mocha/bin/mocha --require ts-node/register ./test/test-*.ts"
+    "build": "node ./node_modules/webpack/bin/webpack.js",
```

Comment:
Actually @jefff (the other collaborator on this repo) mentioned that to me yesterday. Seems fine to me in that case.